### DISCUSSION TITLE
Expose `hovered` prop in style api for `Select`

### DISCUSSION THREAD

**Discussion by pdevito3** [mantinedev/mantine#2269] [Feature requests]
Similar to how the `Autocomplete` component exposes a `hovered` property, it would be nice if the `Select` component did as well

**Comment by rtivital** [comment]
It is an issue with Autocomplete, all outdated selectors will be removed in next patch. You can use `data-hovered` attribute to apply styles to hovered items `<Select styles={{ item: { '&[data-hovered]': { backgroundColor: 'red' } } }} />`. This will work with `styles` prop, `createStyles` function and any `.css` files.

**Reply by pdevito3** [reply]
Seems to work! Is there a similar way to style the selected item in the dropdown? not seeing an attribute for it in the docs

**Reply by rtivital** [reply]
`data-selected` – https://mantine.dev/core/select/#styles-api
